src/App.js

```diff
@@ -1,25 +1,41 @@
-import React from "react";
-import logo from "./logo.svg";
+import React, { useState } from "react";
+import { Switch, Route, BrowserRouter } from "react-router-dom";
 import "./App.css";
+import List from "./List/List";
+import AddItem from "./AddItem/AddItem";
+import Footer from "./Footer/Footer";
+import Header from "./Header/Header";
 
 function App() {
+  const [isListSelected, setIsListSelected] = useState(true);
+  const [isAddSelected, setIsAddSelected] = useState(false);
+
+  const selectListHandler = () => {
```

Comment:
Feel free to use `function selectListHandler` here instead of `const selectListHandler`!

It's totally an opinion thing, but I find that it's easier to grok code that separates variables with `const`/`let` and functions with `function`. Ultimately this is a code style thing so I won't bring it up again, but I just wanted to share with you my perspective. 😄

Reply:
Just for a difference in opinion -- I disagree with this & like using const for everything. Either is fine, though! 😄

Reply:
I've never seen either of those in use, is it from React Hooks? Learning something new everyday!

Reply:
Yeah, Lines 10 & 11 are using the useState hook to set state, so we can just use functional components instead. :-)